corfudb-tools/src/main/java/org/corfudb/browser/CorfuStoreBrowser.java

```diff
@@ -496,4 +496,36 @@ private void printStreamTagMap(String tag, List<TableName> tables) {
         formatMapping = formatMapping.substring(0, formatMapping.length() - 2);
         log.info("tag: <'{}'> --- Tables[total={}] :: {{}}", tag, tables.size(), formatMapping);
     }
+
+    /**
+     * List all known corfu streams with their names and ids in a cluster.
+     *
+     * @return stream tags
+     * */
+    public Set<String> listAllCorfuStreams() {
+        Map<UUID, String> streamTags = new HashMap<>();
```

Comment:
nit - can we call it something else.. streamInfoMap?

Reply:
done thanks